kolibri/plugins/facility/assets/src/routes.js

```diff
@@ -124,4 +127,19 @@ export default [
       }
     },
   },
+  {
+    path: '/:facility_id?/:device_id?/managesync',
```

Comment:
If I am understanding correctly, `ManageSyncSchedule` is showing the list of all scheduled syncs for this facility, so we don't need the `device_id` parameter here at all:
`path: '/:facility_id?/managesync',`